hivemq-edge/src/frontend/src/extensions/datahub/components/controls/DesignerToolbox.tsx

```diff
@@ -0,0 +1,147 @@
+import { FC, useState } from 'react'
+import { Panel } from 'reactflow'
+import { useTranslation } from 'react-i18next'
+import { motion } from 'framer-motion'
+import {
+  Box,
+  ButtonGroup,
+  HStack,
+  Icon,
+  IconButton,
+  Stack,
+  Step,
+  StepIndicator,
+  StepNumber,
+  Stepper,
+  StepStatus,
+  StepTitle,
+  Text,
+  useDisclosure,
+  useSteps,
+  VStack,
+} from '@chakra-ui/react'
+import { FaAngleLeft, FaAngleRight, FaTools } from 'react-icons/fa'
+import { MdSkipNext, MdSkipPrevious } from 'react-icons/md'
```

Comment:
note(to:@h2xd, non-blocking): Let's chat on Tuesday on how we can align icons within all products